api/libproject/devstart.rb

```diff
@@ -1798,12 +1830,19 @@ def setup_project_data(gcc, cdr_db_name, public_db_name)
   # Don't delete the credentials created here; they will be stored in GCS and reused during
   # deployment, etc.
   with_cloud_proxy_and_db(gcc) do
-    common.status "Copying service account key to GCS..."
+    common.status "Copying GSuite service account key to GCS..."
     gsuite_admin_creds_file = Tempfile.new("gsuite-admin-sa.json").path
     common.run_inline %W{gcloud iam service-accounts keys create #{gsuite_admin_creds_file}
         --iam-account=gsuite-admin@#{gcc.project}.iam.gserviceaccount.com --project=#{gcc.project}}
     common.run_inline %W{gsutil cp #{gsuite_admin_creds_file} gs://#{gcc.project}-credentials/gsuite-admin-sa.json}
 
+    common.status "Copying FireCloud service account key to GCS..."
+    firecloud_admin_creds_file = Tempfile.new("firecloud-admin-sa.json").path
```

Comment:
Is this secret? Can you make sure it is blocked from being committed by git secrets if so?

Reply:
Yes – it's a secret in the same way all of our service account keys are secrets. IAFAICT the git-secrets config that we apply as part of the prepare-commit-msg hook (https://github.com/all-of-us/utils/blob/master/hooks/apply-git-secrets.sh#L17) should cover all Google-style JSON service account keys.